ostree-upload-server.py

```diff
@@ -111,7 +112,7 @@ def upload(self):
         Receive a flatpak bundle
         """
         if not self._authenticator.authenticate(request):
-            return self._request_authentication()
+            return self.__class__.request_authentication()
```

Comment:
This is more of a thing I recently started doing to support seamless refactorings. The code you were expecting to see would break if the parent class name changed but this code would adapt. I can change it to the more-common style if you want.

Reply:
Not a big deal. I just thought that the linter was suggesting it based on your commit message.